rundeckapp/src/main/groovy/org/rundeck/app/data/providers/storage/GormStorageDataProvider.groovy

```diff
@@ -72,7 +72,7 @@ class GormStorageDataProvider implements StorageDataProvider {
         storage.lastUpdated = new Date()
 
          try {
-             storageDataService.save(storage)
+             storage.save(flush: true)
```

Comment:
I'd put this inside StorageDataService as a custom method, like it was done here: https://github.com/rundeckpro/rundeckpro/pull/2848/commits/e45b4fae6bc3dd7603e1cd61f8e0c850e18093c3

Reply:
This change will be deferred for a later version